### PR TITLE
Update 09-Simple_Rna-Seq_pipeline.md

### PR DESCRIPTION
**Pull request by ggrimes**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

File: _episodes/09-Simple_Rna-Seq_pipeline.md [hunk 1]
```diff
@@ -661,12 +741,12 @@ process MULTIQC {
...
-Channel
-    .fromFilePairs( params.reads, checkIfExists:true )
-    .set { read_pairs_ch }
 
 workflow {
-  index_ch=INDEX(params.transcriptome)
+  read_pairs_ch = Channel.fromFilePairs( params.reads, checkIfExists:true )
+  transcriptome_ch = Channel.fromPath( params.transcriptome, checkIfExists:true )
+
+  index_ch=INDEX(transcriptome_ch)
```
Comment (mahesh-panchal) [inline _episodes/09-Simple_Rna-Seq_pipeline.md:749]:
```suggestion
  index_ch = INDEX( transcriptome_ch )
```

File: files/scripts/rnaseq_pipeline/script4.nf [hunk 2]
```diff
@@ -42,7 +41,7 @@ process INDEX {
 process QUANT {
 
     input:
-    path index
+    each index
```
Comment (mahesh-panchal) [inline files/scripts/rnaseq_pipeline/script4.nf:44]: Why change this? This should be a path.

File: files/scripts/rnaseq_pipeline/script4.nf [hunk 3]
```diff
@@ -55,9 +54,9 @@ process QUANT {
 }
 
 workflow {
-    Channel.fromFilePairs( params.reads, checkIfExists:true )
-        .set { read_pairs_ch }
+   read_pairs_ch = Channel.fromFilePairs( params.reads, checkIfExists:true )
+   transcriptome_ch = Channel.fromPath( params.transcriptome, checkIfExists:true )
```
Comment (mahesh-panchal) [inline files/scripts/rnaseq_pipeline/script4.nf:58]: If you want to make it a value channel, do:
```suggestion
   transcriptome_ch = file( params.transcriptome, checkIfExists:true )
```
Then you do not need the `each` qualifier.

**Review by mahesh-panchal — APPROVED** [review]
I think there are other changes that are needed here, but I'll make my own pull request for it. 
One thing to change however is the input to `QUANT`. The learner should learn to use a file object to make a value channel input instead of using `each`.

File: files/scripts/rnaseq_pipeline/script4.nf (hunk 3 above)
Comment (ggrimes) [reply files/scripts/rnaseq_pipeline/script4.nf:58]: I didn't know you could input a file into a value channel. Your solution makes more sense.

**Pull request merged**